telemetry/definitions/commonDefinitions.json

```diff
@@ -3890,8 +3892,8 @@
                     "required": true
                 },
                 {
-                    "type": "reason",
-                    "required": false
```

Comment:
See first comment at very top.

Reply:
I think we are trying to keep `result` and `reason` as part of our new standard. What is the goal here from removing this?

Reply:
Added them back.